### PR TITLE
feat(engine): add builtin action PushBuildInfo

### PR DESCRIPTION
**Pull request by sguiheux**
1. Description
1. Related issues
1. About tests
1. Mentions

@ovh/cds

### CODE REVIEW

**Review by fsamin — APPROVED** [review]

**Review by richardlt — DISMISSED** [review]

File: engine/worker/internal/action/builtin_build_info.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	wk.SendLog(ctx, workerruntime.LevelInfo, fmt.Sprintf("# Plugin %s v%s is ready", manifest.Name, manifest.Version))
+
+	query := integrationplugin.RunQuery{
+		Options: sdk.ParametersToMap(wk.Parameters()),
+	}
+
+	res, err := integrationPluginClient.Run(ctx, &query)
+	if err != nil {
+		integrationPluginClientStop(ctx, integrationPluginClient, done, stopLogs)
+		return sdk.Result{}, fmt.Errorf("error deploying application: %v", err)
```
Comment (yesnault) [inline engine/worker/internal/action/builtin_build_info.go:79]:
```suggestion
		return sdk.Result{}, fmt.Errorf("error while running integration plugin: %v", err)
```

File: sdk/exportentities/step.go [hunk 2]
```diff
@@ -329,6 +334,7 @@ type Step struct {
 	// step specific data, only one option should be set
 	StepCustom       `json:"-" yaml:",inline"`
 	Script           interface{}           `json:"script,omitempty" yaml:"script,omitempty" jsonschema:"oneof_type=string;array,oneof_required=actionScript" jsonschema_description:"Script.\nhttps://ovh.github.io/cds/docs/actions/builtin-script"`
+	PushBuildInfo    *StepPushBuildInfo    `json:"pushBuildInfo,omitempty" yaml:"pushBuildInfo,omitempty" jsonschema:"oneof_required=actionPushBuildInfo" jsonschema_description:"Push build info.\nhttps://ovh.github.io/cds/docs/actions/builtin-push-build-info"`
```
Comment (yesnault) [inline sdk/exportentities/step.go:337]: md files in another PR?

**Review by yesnault — CHANGES_REQUESTED** [review]

File: sdk/exportentities/step.go (hunk 2 above)
Comment (sguiheux) [reply sdk/exportentities/step.go:337]: doc will come later

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16058.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Review by yesnault — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16064.0 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16064.1 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✘  
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔

**Review by richardlt — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report build-all-cds#16064.3 ✘  
* Build
  * Build and Package All ✔ 
  * Build Debpacker ✔ 
  * Build OS-Ansible-Inventory ✔ 
  * Build SMTPMock ✔ 
  * Lint UI Translate ✔ 
  * Test CLI ✔ 
  * Test Engine ✔ 
  * Test SDK ✔ 
  * Test SDK Rust ✔ 
  * Test UI ✔ 
* Package
  * Docker ✘

**Pull request merged**